### ISSUE TITLE
Combine threshold for Processing and Production job type

### ISSUE THREAD

**Issue by ticoann** [dmwm/WMCore#7118]
Since sites don't distinguish these types, it makes more sense to combined threshold for Processing and Production jobs. However, need to check what would be the side effects.
(Need to prevent MC jobs taking over all the sites)

**Comment by amaltaro** [comment]
Nope, we actually want to keep the same threshold value (not sum them up). So there should be no side effect.

**Comment by ticoann** [comment]
Yes, Alan. I don't mean add them up. :smile:, I meant use only one type for Production and Processing. About the value, shouldn't we just match with CPU bound for site % num of agents?

**Comment by amaltaro** [comment]
@ericvaandering which job type should we keep in the code? Production or Processing?

Seangchan, I'm not sure I got your question. You mean CPUBound pending thresholds should consider the number of agents connected to the same team? If so, then it does already.

**Comment by amaltaro** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Combine threshold for Processing and Production job type](https://trello.com/c/GWGSkRwL/11-combine-threshold-for-processing-and-production-job-type)
